Formula/l/lexido.rb

```diff
@@ -0,0 +1,21 @@
+class Lexido < Formula
+  desc "Innovative assistant for the command-line"
+  homepage "https://github.com/micr0-dev/lexido"
+  url "https://github.com/micr0-dev/lexido.git", tag: "v1.3.1", revision: "36f02d699f2709f0bd51a5f119e0670845b4ad9f"
```

Comment:
Git clones are heavy so we don't do that unless necessary.

```suggestion
  url "https://github.com/micr0-dev/lexido/refs/tags/v1.3.1.tar.gz"
  sha256 "1b0682f512096a27fbf091d5aa7effc4fd71c5ed28646574fb2fb62761338546"
```

Reply:
Will fix when in the next commit thank you.

Reply:
I've just tried to do this on the formula and it fails to download because there are 4 versions of Lexido 
Depending on what system you're on is the one you download. If you build the whole git repo it will build the right one for your system and 
on Mac the user needs to build it from the git 

I'm new to brew so I have no idea what im doing tbh I just read the cookbook and went with it but if there's a way I can get it to work im open to help.

Reply:
We're not pulling one of the prebuilt binaries at https://github.com/micr0-dev/lexido/releases/tag/v1.3.1. This is the URL to the source code archive ("Source code (tar.gz)" on the release page). Every piece of software here needs to be built from source.

Reply:
> We're not pulling one of the prebuilt binaries at https://github.com/micr0-dev/lexido/releases/tag/v1.3.1. This is the URL to the source code archive ("Source code (tar.gz)" on the release page). Every piece of software here needs to be built from source.

I'm getting a 404 error

HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source lexido
==> Auto-updating Homebrew...
Adjust how often this is run with HOMEBREW_AUTO_UPDATE_SECS or disable with
HOMEBREW_NO_AUTO_UPDATE. Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).
==> Fetching lexido
==> Downloading https://github.com/micr0-dev/lexido/refs/tags/v1.3.1.tar.gz
curl: (56) The requested URL returned error: 404                                                                                                                                                                      

Error: lexido: Failed to download resource "lexido"
Download failed: https://github.com/micr0-dev/lexido/refs/tags/v1.3.1.tar.gz

I am maintaining the project on AUR and the link I used for that is 

https://github.com/micr0-dev/lexido/archive/refs/tags/v1.3.1.tar.gz

I tested that one and it worked so I will use that link